app/components/publications/publication/case/contact-persons-panel.js

```diff
@@ -0,0 +1,77 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @argument {PublicationFlow} publicationFlow include: contact-persons,contact-persons.person
+ */
+export default class PublicationsPublicationCaseContactPersonsPanelComponent extends Component {
+  @service store;
+
+  @tracked isOpenAddModal = false;
+
+  get contactPersons() {
+    const contactPersons = this.args.publicationFlow.contactPersons.toArray();
+    // .get() because await is not supported by sort
+    contactPersons.sort((it1, it2) => {
+      const lastName1 = it1.get('person').get('lastName')
+        .toLowerCase();
+      const lastName2 = it2.get('person').get('lastName')
+        .toLowerCase();
+      return lastName1 < lastName2 ? -1 : 1;
+    });
+    return contactPersons;
+  }
+
+  @action
+  openAddModal() {
+    this.isOpenAddModal = true;
+  }
+
+  @action
+  closeAddModal() {
+    this.isOpenAddModal = false;
+  }
+
+  @action
+  async save(contactPersonProperties) {
+    const {
+      firstName,
+      lastName,
+      email,
+      organization,
+    } = contactPersonProperties;
+
+    if (organization?.isNew) {
+      await organization.save();
+    }
+
+    const person = this.store.createRecord('person', {
+      firstName: firstName,
+      lastName: lastName,
+      organization: organization,
+    });
+    await person.save();
+
+    const publicationFlow = this.args.publicationFlow;
+    const contactPerson = this.store.createRecord('contact-person', {
+      email: email,
+      person: person,
+      publicationFlow: publicationFlow,
+    });
+    await contactPerson.save();
+
+    this.isOpenAddModal = false;
+  }
+
+  @action
+  async delete(contactPerson) {
+    const publicationFlow = this.args.publicationFlow;
+    publicationFlow.contactPersons.removeObject(contactPerson);
```

Comment:
Deze `removeObject` is volgens mij niet noodzakelijk. Als de inverse relaties correct gedefinieerd zijn gaat Ember Data na de `contactPerson.destroyRecord` zelf detecteren dat deze array geupdate moet worden.